wire-gradle-plugin/src/test/projects/kotlin-project-kotlin-protos/build.gradle

```diff
@@ -7,12 +7,14 @@ plugins {
 mainClassName = 'com.squareup.dinosaurs.Sample'
 
 repositories {
-  mavenLocal()
+  maven {
+    url "file://${projectDir.absolutePath}/../../../../../build/localMaven"
+  }
   mavenCentral()
 }
 
 dependencies {
-  implementation "com.squareup.wire:wire-runtime:$VERSION_NAME"
+  implementation "com.squareup.wire:wire-runtime-jvm:$VERSION_NAME"
```

Comment:
Can alternatively depend on `wire-runtime` and build fixtures with Kotlin plugin